Essentials/src/com/earth2me/essentials/Essentials.java

```diff
@@ -742,31 +742,38 @@ public Methods getPaymentMethod() {
 
     @Override
     public int broadcastMessage(final String message) {
-        return broadcastMessage(null, null, message, true);
+        return broadcastMessage(null, null, message, true, Collections.emptySet());
     }
 
     @Override
     public int broadcastMessage(final IUser sender, final String message) {
-        return broadcastMessage(sender, null, message, false);
+        return broadcastMessage(sender, null, message, false, Collections.emptySet());
+    }
+
+    @Override
+    public int broadcastMessage(final IUser sender, final String message, final Collection<IUser> excluded) {
+        return broadcastMessage(sender, null, message, false, excluded);
     }
 
     @Override
     public int broadcastMessage(final String permission, final String message) {
-        return broadcastMessage(null, permission, message, false);
+        return broadcastMessage(null, permission, message, false, Collections.emptySet());
     }
 
-    private int broadcastMessage(final IUser sender, final String permission, final String message, final boolean keywords) {
+    private int broadcastMessage(final IUser sender, final String permission, final String message, final boolean keywords, final Collection<IUser> excluded) {
```

Comment:
nit: Consider changing this to instead accept a `Function<IUser, Boolean>` - reason being is that in most cases (including this one) its overkill to pass in a collection with a bunch of additional logic.

Reply:
I've updated the method to use a `Predicate<IUser>` instead of a collection.